index.html

```diff
@@ -40,6 +40,200 @@ <h2 id="green">I'm a software developer</h2>
+                        <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                            don't hesitate to contact me.
+                        </p>
+                    </div>
+                    <a class='resume' href="#">Get my Resume</a>
+                </div>
+                <div class="aboutme-grid">
+                    <!--languages card-->
+                    <div class="aboutme-item">
+                        <div class="item-head">
+                            <img src="./media/languages-icon.svg" alt="languages-icon">
+                        </div>
+                        <div class="title-tags">
+                            <h4>Languages</h4>
+                            <div class="tags">
+                                <ul>
+                                    <li>Javascript</li>
+                                    <li>Ruby</li>
+                                    <li>HTML</li>
+                                    <li>CSS</li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                    <!--Frameworks card-->
+                    <div class="aboutme-item">
+                        <div class="item-head">
+                            <img src="./media/frameworks-img.png" alt="frameworks-icon">
+                        </div>
+                        <div class="title-tags">
+                            <h4>Frameworks</h4>
+                            <div class="tags">
+                                <ul>
+                                    <li>React.js</li>
+                                    <li>Ruby on Rails</li>
+                                    <li>RSpec</li>
+                                    <li>Capybara</li>
+                                    <li>Selenium</li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                    <!--Skills card-->
+                    <div class="aboutme-item">
+                        <div class="item-head">
+                            <img src="./media/skills-img.png" alt="skills-icon">
+                        </div>
+                        <div class="title-tags">
+                            <h4>Skills</h4>
+                            <div class="tags skills">
+                                <ul>
+                                    <li>Database Management</li>
+                                    <li>Version Control</li>
+                                    <li>CLI</li>
+                                    <li>Web Development</li>
+                                    <li>API Design</li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
```

Comment:
- [ ] Kindly make sure to list the technology categories ("Languages", "Frameworks" and "Skills") container using the `ul` tag and make sure each one is wrapped into the `li` tag.

Reply:
Hello, @levy002, thanks for reviewing my code. I have a doubt, do I need to make <h4> tags("Languages", "Frameworks", "Skills") as list elements or make the whole card(each card) as a list element? kindly suggest what I can do. 
Thank you.

Reply:
Hello, @levy002 I understood the change specified. you made it clear in the sent mail I just had to read it again. thanks again.

Reply:
Thanks @tan12082001 :+1: